tests/conftest.py

```diff
@@ -123,6 +123,13 @@ def update_metadata_callback(req):
     "DELETE::api/v1/samples/761bc54b97f64980": {},
     "GET::api/v1/samples/public": {},
     "GET::api/v1/projects/public": {},
+    "GET::api/v1/analyses/public": [{
```

Comment:
OK, sorry I didn't look at this more recently. We need to update this mock data to look like an actual analysis – this is got a hodgepodge of `analysis` and `job` fields. Here's one for reference:

```
{
    "$uri": "/api/v1/analyses/de58fb3db76c42f3",
    "analysis_type": "classification",
    "complete": true,
    "created_at": "2017-08-03T17:23:21.931982+00:00",
    "error_msg": null,
    "job": {
        "$uri": "/api/v1/jobs/cc1d331e1ee54bac",
        "analysis_type": "classification",
        "created_at": "2016-05-05T17:27:02.116480+00:00",
        "name": "One Codex Database",
        "public": true
    },
    "sample": {
        "$uri": "/api/v1/samples/205bef56b3d6457d",
        "created_at": "2017-08-03T17:23:20.284145+00:00",
        "filename": "MSA-1000.16S.example.fastq.gz",
        "metadata": {
            "$ref": "/api/v1/metadata/ab296c81940e4cc0"
        },
        "owner": {
            "$ref": "/api/v1/users/5891ee65711c4d5e"
        },
        "primary_classification": {
            "$ref": "/api/v1/classifications/980c7388c54a46b8"
        },
        "project": {
            "$ref": "/api/v1/projects/66477c4c379a41a5"
        },
        "size": 9342972,
        "tags": [
            {
                "$ref": "/api/v1/tags/e2910f93ec154617"
            },
            {
                "$ref": "/api/v1/tags/7dd2d4ae9e9c4f14"
            },
            {
                "$ref": "/api/v1/tags/7d350965f74346c1"
            },
            {
                "$ref": "/api/v1/tags/1638c7a570214fda"
            }
        ],
        "visibility": "private"
    },
    "success": true
}
```